src/js/tabs/osd.js

```diff
@@ -394,7 +397,8 @@ OSD.constants = {
     ],
     TIMER_PRECISION: [
         'SECOND',
-        'HUNDREDTH'
+        'HUNDREDTH',
+        'TENTH'
```

Comment:
Hmm, would it make sense to make these translatable? @McGiverGim what do you think?

Reply:
All can be translatable. I think is a good idea. And maybe at some moment all the OSD elements too.